src/v4l/mod.rs

```diff
@@ -0,0 +1,270 @@
+///
+/// # Neolink V4l
+///
+/// This module serves the v4l streams for the
+/// `neolink v4l` subcommand
```

Comment:
I object slightly to `v4l` as a name. Presumably Windows users would like something like this too and a hypothetical implementation using native Windows APIs would not be named `v4l`. What's a good name for the _feature_, not the mechanism?

Reply:
Just as a reference: Macos has CoreMediaIO and windows has DirectShow for virtual camera. Neither of which seem to have any high level crates, it's just an sys level FFI stuff at the moment, we'd have to roll it ourselves.

We could perhaps get similar functionality though with just a fifo exposing the current streams.

I do kinda agree that the name should be generic rather than `v4l` (video 4 linux). Maybe `virtual_camera`?

Reply:
`virtcam`?